models/src/main/scala/coop/rchain/casper/protocol/CasperMessage.scala

```diff
@@ -337,13 +337,13 @@ final case class ProcessedDeploy(
 
 object ProcessedDeploy {
   def empty(deploy: Signed[DeployData]) =
-    ProcessedDeploy(deploy, PCost(), List.empty, isFailed = false)
+    ProcessedDeploy(deploy, PCost(), Vector.empty, isFailed = false)
   def from(pd: ProcessedDeployProto): Either[String, ProcessedDeploy] =
     for {
       ddProto   <- pd.deploy.toRight("DeployData not available")
       dd        <- DeployData.from(ddProto)
       cost      <- pd.cost.toRight("Cost not available")
-      deployLog <- pd.deployLog.toList.traverse(Event.from)
+      deployLog <- pd.deployLog.toVector.traverse(Event.from)
```

Comment:
deployLog on the right comes from ProcessedDeployProto which defines it to be a Seq. That is why toVector is necessary so that it is converted from Seq to Vector. Maybe we make deployLog in ProcessedDeployProto a vector instead of Seq as well?